src/components/Footer.js

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2016 OpenMarket Ltd
+Copyright 2019 Matrix.org Foundation C.I.C.
```

Comment:
This is not right. These old files *are* copyrighted by OpenMarket. What I meant by my comment is for any new code you introduce you should be using the copyright of your Matrix C.I.C. as you do not work for OpenMarket.